test/test_utils/sagemaker.py

```diff
@@ -196,11 +196,14 @@ def install_sm_local_dependencies(framework, job_type, image, ec2_conn):
         # TF inference test fail if run as soon as instance boots, even after health check pass. rootcause:
         # sockets?/nginx startup?/?
         install_custom_python("3.6", ec2_conn)
+    if framework == "huggingface_tensorflow":
+        install_custom_python("3.7", ec2_conn)
```

Comment:
This statement doesn't link python3 to python3.7. This is a fix that needs to go into the `install_custom_python` function.